.opencode/skills/testing/SKILL.md

```
@@ -11,20 +11,16 @@ After making changes you **must always** evaluate and test them.
-Use module-graph script to find which hosts and homes are affected by changed files:
+Use module-graph script with `--since` to find only hosts and homes affected by files changed since commit or ref. Add `--refine` to further narrow results for files under `modules/nixos/` and `modules/home-manager/` when module exposes `options.<path>.enable`:
 
 ```bash
-./flake/dev/scripts/module-graph.nu
+SILENT=true nix run .#module-graph -- --since <COMMIT_HASH> --refine --report
 ```
 
-This outputs JSON showing which configs use each file:
+This will output a report similar to the following, showing which hosts and homes are affected by the changes:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)